mobile/src/bundles/common/components/input/input.tsx

```diff
@@ -26,6 +30,8 @@ const Input = <T extends FieldValues>({
     label,
     name,
     placeholder,
+    type,
+    ...properties
```

Comment:
```suggestion
    editable,
    ...props
```
Merge the latest development to your branch and there'll not be an error